odk1-src/form-audit-log.rst

```diff
@@ -0,0 +1,146 @@
+.. spelling::
+
+  timestamp
+
+Logging enumerator behavior
```

Comment:
I would add a reference to /form-widgets/#metadata-widgets so people know that they can timestamp in this way.

Reply:
Do you mean a reference from there?

Reply:
Yes a reference from metadata-widgets pointing to this document.